<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindowClass</class>
 <widget class="QMainWindow" name="CMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>167772</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>网络节点布局系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>保存</string>
     </property>
     <addaction name="savePic"/>
     <addaction name="saveDoc"/>
    </widget>
    <addaction name="newFile"/>
    <addaction name="openFile"/>
    <addaction name="menu_2"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="fishEyes"/>
    <addaction name="lableDisplay"/>
    <addaction name="fullScreen"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>运行</string>
    </property>
    <addaction name="beginLayout"/>
    <addaction name="stopLayout"/>
    <addaction name="continueLayout"/>
    <addaction name="relayout"/>
   </widget>
   <widget class="QMenu" name="menuTu">
    <property name="title">
     <string>图序列</string>
    </property>
    <addaction name="nodesDegreeEdit"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuTu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>打开节点分布文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="savePic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
   <property name="toolTip">
    <string>存下布局图像</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveDoc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>保存文档</string>
   </property>
   <property name="toolTip">
    <string>文档形式保存布局结果</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="exit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="beginLayout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>开始布局</string>
   </property>
   <property name="toolTip">
    <string>开始进行3维布局</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="stopLayout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>暂停布局</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="algoriParamSeting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>算法参数设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="continueLayout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>继续布局</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="fishEyes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>鱼眼效果</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="relayout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>重新布局</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="fullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>全屏/正常</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="openFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="nodesDegreeEdit">
   <property name="text">
    <string>节点度编辑</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+E</string>
   </property>
  </action>
  <action name="lableDisplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>显示/隐藏标签</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
